Jenkinsfile

```diff
@@ -30,7 +32,11 @@ devToolsProject.run(
       black: { data.venv.run('black --check .') },
       flake8: { data.venv.run('flake8 -v') },
       groovylint: { groovylint.check('./Jenkinsfile') },
-      molecule: { data.venv.run('molecule --debug test --all') },
+      molecule: {
+        withEnv(['ANSIBLE_ROLES_PATH=.ansible/roles']) {
+          data.venv.run('molecule --debug test')
```

Comment:
The `--all` flag is gone. Is it intended?

Reply:
Nope, sorry.